File: PaylocityChallenge/Paylocity.Models/Models/Benefit.cs
using Paylocity.Common.Interfaces;

namespace Paylocity.Models.Models
{
    public class Benefit : IUniqueModel
    {
        public int Id { get; set; }
        public decimal CostPerEmployeePerYear { get; set; }
        public decimal CostPerDependentPerYear { get; set; }
    }
}

File: PaylocityChallenge/paylocity-frontend/src/models/Benefit.ts
export default class Benefit {
    private _id: number;
    private _costPerEmployeePerYear: number;
    private _costPerDependentPerYear: number;

    constructor() {
        this._id = 0;
        this._costPerEmployeePerYear = 0;
        this._costPerDependentPerYear = 0;
    }

    get id(): number {
        return this._id;
    }

    get costPerEmployeePerYear(): number {
        return this._costPerEmployeePerYear;
    }

    get costPerDependentPerYear(): number {
        return this._costPerDependentPerYear;
    }
}